### PR TITLE
magento/devdocs#: Layout instructions. Adding the example of disabling plugin.

### PR DESCRIPTION
**Pull request by DenisSaltanahmedov**
## Purpose of this pull request

This pull request (PR) Adding example of disabling plugins.
The plugin topic provides information that each plugin can be disabled a few times, but the document still does not have any example, I think it will be good to have a separate section in the document with the instruction and the example of disabling a plugin.

## Affected DevDocs pages
- https://devdocs.magento.com/guides/v2.3/extension-dev-guide/plugins.html

## Links to Magento source code
-  https://github.com/magento/devdocs/blob/master/src/guides/v2.3/extension-dev-guide/plugins.md

whatsnew
Added the 'Disabling Plugins' section to the [Plugin (Interceptors)](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/plugins.html) topic.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/extension-dev-guide/plugins.md [hunk 1]
```diff
@@ -473,6 +473,20 @@ Magento uses plugins defined in the global scope when the system is in a specifi
 
 For example, the developer can disable a global plugin in the [backend](https://glossary.magento.com/backend) area by disabling it in the specific `di.xml` file for the backend area.
 
+### Disabling plugin.
+
+Each plugin can be disabling in a ``di.xml`` file. For it the ``disabled`` element should be set to ``true``
```
Comment (rogyar) [inline src/guides/v2.3/extension-dev-guide/plugins.md]:
```suggestion
Each plugin can be disabled in a ``di.xml`` file. To disable a plugin the ``disabled`` parameter of the plugin declaration should be set to ``true``
```

File: src/guides/v2.3/extension-dev-guide/plugins.md [hunk 2]
```diff
@@ -473,6 +473,20 @@ Magento uses plugins defined in the global scope when the system is in a specifi
...
+
+For example:
+
+```xml
+<type name="Magento\Checkout\Block\Checkout\LayoutProcessor">
+    <plugin name="ProcessPaymentConfiguration" disabled="true"/>
+</type>
+```
+
+Where ``ProcessPaymentConfiguration`` is the name plugin declared in the `vendor/magento/module-payment/etc/frontend/di.xml` and is require parameter.
```
Comment (rogyar) [inline src/guides/v2.3/extension-dev-guide/plugins.md]:
```suggestion
Where ``ProcessPaymentConfiguration`` is the name of the plugin declared in the `vendor/magento/module-payment/etc/frontend/di.xml`.
```

**Comment by DenisSaltanahmedov** [comment]
Hi @rogyar, thanks for your review.
I have fixed the recommendations.

**Pull request merged**